requirements.txt

```diff
@@ -4,10 +4,6 @@
 #
 #    pip-compile requirements.in
 #
-appnope==0.1.2
```

Comment:
General question for my own curiosity: was this due to someone previously freezing the dependencies on macOS (and this not being needed on Linux)?

Reply:
Hmm, possibly. Or it just dropped it as a hard dependency?

I have a WIP that changes this to build the requirements.txt using the docker image, rather than the developers machine, which should hopefully fix such issues. And pinning the requirements.in in the process, as God intended.